lib/src/ast.dart

```diff
@@ -1,15 +1,25 @@
+/// A class that describes the position of the source text.
+class SourcePosition {
+  const SourcePosition(this.start, this.end);
+
+  /// The start position of this query, exclusive.
+  final int start;
+
+  /// The end position of this query, exclusive.
```

Comment:
To be honest, i'm not sure. I copied it from https://pub.dev/documentation/source_span/latest/source_span/SourceLocation-class.html

Reply:
Ah, sorry, I put the comment at the wrong code line: I meant to ask whether the `start` is inclusive, instead of the current documention stating exclusive.

Reply:
Fixed